docs/hosted-orchestration/building-images.md

```diff
@@ -0,0 +1,168 @@
+---
+id: building-images
+title: Building Docker Images
+sidebar_label: Building Images
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This document describes how to create and use custom Docker images with Sauce Labs.
+
+## Overview
+Sauce Labs supports Docker, providing you with a powerful way to orchestrate tests. In order to use Hosted Orchestration you will need to package your test code and all of its dependencies as a Docker Image and publish it to a Docker container regisry.
+
+## System Requirements
+
+You can run build images locally via Docker ([Installation Requirements](https://docs.docker.com/engine/install/#supported-platforms)) or as part of your CI process. System requirements vary depending on your intention so the following reference serves as a general guide:
+
+## Building and Pushing Images
+
+### Prerequisites
+
+- A working Docker installation. For more information see [Docker's Getting Started documentation](https://docs.docker.com/get-started/).
+
+### Creating a Dockerfile
+
+To create a custom image, you must create a
+Dockerfile
+. This is a text document containing commands that Docker uses to assemble an image.
```

Comment:
Linking to official Docker documentation on Dockerfile would be helpful here

Reply:
Maybe even a basic sample Dockerfile here. I think Python would be a good choice.

```
FROM python:3
WORKDIR /demo-python
COPY test-requirements.txt /demo-python
RUN pip install -r requirements.txt
COPY ./tests/ /demo-python/tests/
```